aspnet/conceptual-overview/aspnet.rst

```diff
@@ -57,23 +69,23 @@ ASP.NET Core applications are defined using a public ``Startup`` class:
       }
   }
 
-The ``ConfigureServices`` method defines the services used by your application and the ``Configure`` method is used to define what middleware makes up your request pipeline. See :doc:`/fundamentals/startup` for more details.
+- ``ConfigureServices`` defines the services (see **Services** below) used by your app (such as the ASP.NET MVC Core framework, Entity Framework Core, Identity, etc.)
+- ``Configure`` defines the :doc:`middleware </fundamentals/middleware>` in the request pipeline
+- See :doc:`/fundamentals/startup`
 
 Services
 --------
 
-A service is a component that is intended for common consumption in an application. Services are made available through dependency injection. ASP.NET Core includes a simple built-in inversion of control (IoC) container that supports constructor injection by default, but can be easily replaced with your IoC container of choice. See :doc:`/fundamentals/dependency-injection` for more details.
-
-Services in ASP.NET Core come in three varieties: singleton, scoped and transient. Transient services are created each time they’re requested from the container. Scoped services are created only if they don’t already exist in the current scope. For Web applications, a container scope is created for each request, so you can think of scoped services as per request. Singleton services are only ever created once.
+A service is a component that is intended for common consumption in an application. Services are made available through dependency injection. ASP.NET Core includes a simple built-in inversion of control (IoC) container that supports constructor injection by default, but can be easily replaced with your IoC container of choice. In addition to its loose coupling benefit, DI makes services available throughout your app. For example, :doc:`Logging </fundamentals/logging>` is available throughout your app. See :doc:`/fundamentals/dependency-injection` for more details.
 
 Middleware
 ----------
 
-In ASP.NET Core you compose your request pipeline using :doc:`/fundamentals/middleware`. ASP.NET Core middleware perform asynchronous logic on an ``HttpContext`` and then optionally  invoke the next middleware in the sequence or terminate the request directly. You generally "Use" middleware by invoking a corresponding extension method on the ``IApplicationBuilder`` in your ``Configure`` method.
+In ASP.NET Core you compose your request pipeline using :doc:`/fundamentals/middleware`. ASP.NET Core middleware performs asynchronous logic on an ``HttpContext`` and then either invokes the next middleware in the sequence or terminates the request directly. You generally "Use" middleware by invoking a corresponding ``UseXYZ`` extension method on the ``IApplicationBuilder`` in the ``Configure`` method.
 
 ASP.NET Core comes with a rich set of prebuilt middleware:
 
-- :doc:`/fundamentals/static-files`
+- Static files with the :doc:`Static file service </fundamentals/static-files>`
```

Comment:
@danroth27 
That renders as "Static files with the Static file"
My version reads much better don't you think?